randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -630,13 +662,16 @@ Extra - asset_id: collision_camera_045
   * Extra - actor_type: weightactivatedplatform
   > Pickup (Power Bomb Tank)
       Any of the following:
-          Can Spiderspark
+          Spiderspark (Beginner) and Can Spiderspark
           Infinite Bomb Jump (Expert) and Lay Bomb
   > Door to Diggernaut Arena
       Trivial
   > Top Platform
       Any of the following:
-          Space Jump or Single-wall Wall Jump (Expert)
+          Single-wall Wall Jump (Expert)
+          All of the following:
+              Space Jump
+              Screw Attack or Movement (Intermediate)
           All of the following:
               Morph Ball
               Any of the following:
```

Comment:
GH doesn't let me write a comment below this line...
 If I read that correctly it would be enough to have Morph + Spider but with Space Jump you are required to have Screw or Movement (Intermediate). That doesn't sound balanced to me. Isn't Morph + Spider much harder or do the enemies not reach you at the left or right side?

Reply:
Actually Spider should have some movement since it involves dodging the enemies iirc

Reply:
Just tested this. All you have to do is hold up. Might not even need a trick then